src/OpenTK/Platform/MacOS/CocoaNativeWindow.cs

```diff
@@ -720,7 +721,7 @@ public override void ProcessEvents()
                     case NSEventType.MouseMoved:
                         {
                             Point p = new Point(MouseState.X, MouseState.Y);
-                            if (CursorVisible)
+                            if (CursorVisible && !cursorConfined)
```

Comment:
Decoupling Grabbed and Visible, surely this only needs to check cursorConfined now?